src/edu/washington/escience/myria/coordinator/catalog/MasterCatalog.java

```diff
@@ -1073,8 +1079,16 @@ protected Schema job(final SQLiteConnection sqliteConnection) throws CatalogExce
         long numTuples = statement.columnLong(3);
         long queryId = statement.columnLong(4);
         String created = statement.columnString(5);
+        PartitionFunction howPartitioned;
+        try {
+          howPartitioned =
+              MyriaJsonMapperProvider.getMapper().readValue(statement.columnString(6), PartitionFunction.class);
+        } catch (final IOException e) {
+          LOGGER.debug("Error deserializing howPartitioned for dataset #{}", relationKey.toString(), e);
```

Comment:
the `toString()` is unnecessary; it just adds work and raises the overhead when debug is not enabled.